### PR TITLE
Finish audio

### PR DESCRIPTION
**Pull request by ShFil119**

### CODE REVIEW

File: src/audio/AudioManager.cpp [hunk 1]
```diff
@@ -40,19 +40,18 @@
 #include "sampman.h"
 
 cAudioManager &AudioManager = *(cAudioManager *)0x880FC0;
```
Comment (erorcun) [inline src/audio/AudioManager.cpp:42]: why not remove this wrapper :)

Comment (ShFil119) [reply src/audio/AudioManager.cpp:42]: I thought that maybe there's unreversed function (yet) which uses audio.

Comment (erorcun) [reply src/audio/AudioManager.cpp:42]: I just looked to xref's to AudioManager, but ofc you know better in that regard

**Comment by erorcun** [comment]
lgtm

**Pull request merged**